dbms/src/Server/TCPHandler.cpp

```diff
@@ -471,6 +471,7 @@ void TCPHandler::sendHello()
 	writeVarUInt(DBMS_VERSION_MAJOR, *out);
 	writeVarUInt(DBMS_VERSION_MINOR, *out);
 	writeVarUInt(ClickHouseRevision::get(), *out);
+	writeStringBinary(DateLUT::instance().getTimeZone(), *out);
```

Comment:
This is a backwards-incompatible protocol change. Is there a mechanism to avoid breaking interaction with old and clients and servers in this case?